integrations/tensorflow/bindings/python/iree/tf/support/module_utils.py

```diff
@@ -89,12 +89,10 @@ def _get_tf_import_output_kwargs(artifacts_dir: str,
   return kwargs
 
 
-def _incrementally_compile_tf_module(
-    module: Type[tf.Module],
-    backend_info: "BackendInfo",
-    exported_names: Sequence[str] = (),
-    artifacts_dir: Optional[str] = None,
-) -> Tuple[bytes, Optional[str]]:
+def _incrementally_compile_tf_module(module: Type[tf.Module],
```

Comment:
It seems like all that happened here is you removed the return type? Why?

Reply:
It triggered pytype errors that are unrelated to changes in this pull request and that I cannot make sense of. Likely there are issues in pytype?

Added back to show the issue.

Reply:
@phoenix-meadowlark: do you know how to fix it? (Also we don't have this return type hint for `_incrementally_compile_tf_signature_def_saved_model`, which is structurally similar.)

Reply:
I think it is actually related to this PR. output_kwargs used to only be a dict with string values. Now you've added something that's a list of string values. The shared type of those two is `Sequence[str]` (because string is itself a sequence of strings)

Reply:
I'm not sure I follow here. But maybe it's just me not familiar with how pytype works..

 We are returning `immediate_result, output_file`. `output_file` is created via `output_file = output_kwargs.get("output_file")`, which has nothing to do with `output_kwargs["extra_args"]`. Are you saying that for a `dict`, pytype does not look at the specific key, it just uses the most common base type for all values? (We are not seeing `Any` here actually, so I assume it at least scans all values. Then why not look at the specific key..) 

Also do you know why the change for `immediate_result` from `bytes` to `Any`?

Reply:
I know basically nothing about how pytype works. This looks to me a lot like the type inference I'm describing though. It's now inferred that `output_args` is a `Dict[str, Sequence[str]]` which means that `output_args.get` returns a `Sequence[str]`. It's not surprising to me that it doesn't look at the key. The key would be a runtime-dependent value.

I'd further guess that the root cause of the change of `immediate_result` to `Any` is the same. It now thinks that `output_file` is a `Sequence[str]` so it can no longer infer what happens when you open it.

I have no idea whether we care about any of this, just trying to figure out what's going on

Reply:
I would recommend adding an inline comment `# pytype: disable=bad-return-type`. Pytype can be difficult to work with and isn't worth blocking on outside of the rare cases where it finds a bug.